### ISSUE TITLE
Proto build on Windows 10 failing

### ISSUE THREAD

**Issue by sononix** [hyperium/tonic#821]
cargo build failing - upstream builds dependent on tonic currently affected
appears tied to the interop folder - building within the folder also fails 

Build works on Linux but not on window atm

<code>
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "protoc failed: proto/grpc/testing: warning: directory does not exist.\r\nCould not make proto path relative: proto/grpc/testing/test.proto: No such file or directory\r\n" }', interop\build.rs:4:40
stack backtrace:
   0: rust_begin_unwind
             at /rustc/09c42c45858d5f3aedfa670698275303a3d19afa\/library\std\src/panicking.rs:517:5
   1: core::panicking::panic_fmt
             at /rustc/09c42c45858d5f3aedfa670698275303a3d19afa\/library\core\src/panicking.rs:101:14
   2: core::result::unwrap_failed
             at /rustc/09c42c45858d5f3aedfa670698275303a3d19afa\/library\core\src/result.rs:1617:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/09c42c45858d5f3aedfa670698275303a3d19afa\library\core\src/result.rs:1299:23
   4: build_script_build::main
             at C:\Users\ms\CODE\RUST\tonic\interop\build.rs:4:5
   5: core::ops::function::FnOnce::call_once
             at /rustc/09c42c45858d5f3aedfa670698275303a3d19afa\library\core\src\ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

</code>

**Comment by allan2** [comment]
```
Could not make proto path relative: proto/grpc/testing/test.proto: No such file or directory
```
Given this error, is the path correct?

It would help if you provided the code to reproduce the error.

**Comment by sononix** [comment]
Apologises its a basic build - cargo build in the root folder 

navigating  into tonic\interop and running cargo build there generates the error mentioned 
the files and folders exists https://github.com/hyperium/tonic/tree/master/interop/proto/grpc/testing

mind the folder structure within the repo looks a little strange

**Comment by davidpdrsn** [comment]
I don't have access to a windows machine so not sure how to debug this 🤔 I do know from work that tonic works just fine on windows though.

**Comment by LucioFranco** [comment]
We have a CI job that runs windows https://github.com/hyperium/tonic/runs/4038722988 try running the command in that?

**Comment by sononix** [comment]
Thanks @LucioFranco this lead me to the fact that my installation probably wasnt 100% (I think I used choco to install rust) - I couldnt even call rustup 
<code>
rustup component add rustfmt!!!!
</code>

A reinstallation of the binary from the website did the trick - 
https://www.rust-lang.org/tools/install

tks again 
M

**Comment by hemanth94** [comment]
@LucioFranco I have tried reinstalling rust from official page as you suggested. But i'm still facing this error `"protoc failed: Could not make proto path relative: ./protos/format.proto: No such file or directory`. Tried changing different versions of Protoc as well.

I was trying to build this `lance-core v0.9.6 (https://github.com/universalmind303/lance?rev=ffd4ac6ee2c61b3792a904b2e12152b246e837e6#ffd4ac6e)`.

Any help is much appreciated. Thanks.
